NOTICE

```diff
@@ -0,0 +1,226 @@
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at https://3rdpartysource.microsoft.com, or you may

```

Comment:
> you may send a check or money order for US $5.00

Where did this come from? Regardless of the merit of the statement, it's not something that should be in our repo. Perhaps this open source agreement is centralized in a place we can link to?

Reply:
How about we update to:

This software incorporates material from third parties. Microsoft makes certain
open source code available at https://3rdpartysource.microsoft.com, or you may
You may visit the website for any open source inquiries.